### ISSUE TITLE
Decoding failed: Syntax error

### ISSUE THREAD

**Issue by kehskas** [axis/axiscommerce#234]
Создаем товар, заполняем только первую вкладку. Нажимаем сохранить и продолжить - просит заполнить обязательные поля на других вкладках. Заполняем, нажимаем сохранить и продолжить - вылезает ошибка (какая не помню). При повторной попытке сохранить снова ошибка просит сделать SKU уникальным. Меняем SKU - во второму кругу то же самое.

После получаем два товара в категории "без категорий" которые невозможно ни отредактировать ни удалить. При любой попытке что либо сделать ошибка - Decoding failed: Syntax error

2012-12-28T03:12:40+04:00 DEBUG (7): Decoding failed: Syntax error
# -0 /app/code/Axis/Catalog/controllers/Admin/ProductController.php(504): Zend_Json::decode('{\"0\":\"2\"}')
# -1 /library/Zend/Controller/Action.php(516): Axis_Catalog_Admin_ProductController->removeAction()
# -2 /library/Zend/Controller/Dispatcher/Standard.php(295): Zend_Controller_Action->dispatch('removeAction')
# -3 /library/Zend/Controller/Front.php(954): Zend_Controller_Dispatcher_Standard->dispatch(Object(Zend_Controller_Request_Http), Object(Zend_Controller_Response_Http))
# -4 /library/Zend/Application/Bootstrap/Bootstrap.php(97): Zend_Controller_Front->dispatch()
# -5 /library/Zend/Application.php(366): Zend_Application_Bootstrap_Bootstrap->run()
# -6 /index.php(73): Zend_Application->run()
# -7 {main}

**Comment by kehskas** [comment]
В "наборах значений" - такая же ошибка. 

Axis 0.8.7.1 (не обновился до 0.8.7.2 т.к. обновление само по себе порядочный геморой и было сказано что ничего особо критичного изменено не было)

Расскажите как починить.

**Comment by vovayatsyuk** [comment]
Вот эти бекслеши не дают прочитать запрос: Zend_Json::decode('{\"0\":\"2\"}')

Они, скорее всего, добавлены настройкой magic_quotes. Нужно их выключить.

Так как они не выключились с помощью .htaccess (https://github.com/axis/axiscommerce/blob/develop/.htaccess#L37), можно предположить, что у вас
fastCGI режим сервера.

Попробуйте переименовать php.ini.sample в корне Аксиса в php.ini или в php5.ini.

http://axiscommerce.com/forum/products-stock#comment-13452
http://axiscommerce.com/forum/php-magic-quotes#comment-5445

**Comment by kehskas** [comment]
Да. FastCGI - была при установке такая проблема - что-то делал, чтобы
устранить. Видимо хостер опять начудил.
Спасибо!

Кстати могу попробовать взять на себя функции по организации краудфандинга
если самим некогда.

С наступающим!

31 декабря 2012 г., 17:06 пользователь Vova Yatsyuk <
notifications@github.com> написал:

> Вот эти бекслеши не дают прочитать запрос:
> Zend_Json::decode('{\"0\":\"2\"}')
> 
> Они, скорее всего, добавлены настройкой magic_quotes. Нужно их выключить.
> 
> Так как они не выключились с помощью .htaccess (
> https://github.com/axis/axiscommerce/blob/develop/.htaccess#L37), можно
> предположить, что у вас
> fastCGI режим сервера.
> 
> Попробуйте переименовать php.ini.sample в корне Аксиса в php.ini или в
> php5.ini.
> 
> http://axiscommerce.com/forum/products-stock#comment-13452
> http://axiscommerce.com/forum/php-magic-quotes#comment-5445
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/axis/axiscommerce/issues/234#issuecomment-11777084.

## 

Truly yours,
snow guide in the Kola Arctic's - http://snowings.ru/about
